workflow/tests/hpxml_translator_test.rb

```diff
@@ -861,6 +861,12 @@ def _verify_simulation_outputs(runner, rundir, hpxml_path, results)
     clg_energy = results.select { |k, v| (k.include?(': Cooling (MBtu)') || k.include?(': Cooling Fans/Pumps (MBtu)')) && !k.include?('Load') }.map { |k, v| v }.inject(0, :+)
     assert_equal(clg_load_frac > 0, clg_energy > 0)
 
+    # Unmet Load
+    if hpxml_path.include? 'base-hvac-ideal-air'
```

Comment:
Rather than using the file name, which could change in the future, how about using `if htg_load_frac == 0 and clg_load_frac == 0`?